_config.yml

```diff
@@ -1,4 +1,4 @@
 name: Your New Jekyll Site
 markdown: redcarpet
-pygments: true
-baseurl: /figure
+highlighter: null
+baseurl: /figure
```

Comment:
Generally, we should be able to resolve conflicts in `_config.yml` except for the line around the one modified in 239638c, i.e. the `pygments:` line. If we want to make that change permanent, we may have to isolate this line modification in a PR against `gh-pages-staging`, merge the PR, force push `ome/gh-page-staging` to `ome/gh-pages` and apply the CNAME commit on top of that.